husky_components/matrix/main.js

```diff
@@ -42,15 +42,16 @@ define(function() {
 
         toggleIcon: function(event) {
             var $target = event.currentTarget,
-                $tr = this.sandbox.dom.parent($target),
-                $allTargets = this.sandbox.dom.find('span[class^="fa-"]', $tr),
+                $td = this.sandbox.dom.parent($target),
+                $tr = this.sandbox.dom.parent($td),
+                $allTargets = this.sandbox.dom.find('span[class^="fa-"]', $td),
                 $activeTargets,
-                $link = this.sandbox.dom.find('td:last-child span', $tr);
+                $link = this.sandbox.dom.find('td:last-child > span:last-of-type', $tr);
```

Comment:
Don't you also want to use `last-child` here instead of `last-of-type`? Not that we are running in some strange behavior, because we are using another selector above.